### PR TITLE
Add documents for release flow

### PR DESCRIPTION
**Pull request by miseyu**
## Summary
* Add documentation describing the release flow.

### CODE REVIEW

File: docs/ReleaseFlow.md [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+``` yaml
+version: 2.1.0
+```
+
+## Release
+Run the release manually from the Github UI.
+* [Release Action](https://github.com/scalar-labs/helm-charts/actions/workflows/release.yml)
+
+## Points of Attention
+### If the version of the `main` branch of the `charts` you are releasing is higher than the version you are releasing
```
Comment (feeblefakie) [inline docs/ReleaseFlow.md:19]: I think it's simpler to always follow the process. 
What do you think?
It would be great if we can chat about it tomorrow.

**Comment by miseyu** [comment]
@feeblefakie 
I have corrected the points pointed out.
PTAL :bow:
https://github.com/scalar-labs/helm-charts/pull/73/commits/09b815ccbc2f23648a78b1a51959188607097caf

**Review by kota2and3kan — APPROVED** [review]
LGTM. 
I was able to release new version chart, according to these steps!
Thank you!

**Review by feeblefakie — APPROVED** [review]
LGTM!

**Pull request merged**